### PR TITLE
Add more fields and docs

### PR DESCRIPTION
**Pull request by autarch**

### CODE REVIEW

**Comment by autarch** [comment]
This is still a work in progress ...

**Comment by mickeyn** [comment]
cool, so I'll mark it WIP in the title. let me know when you want it merge then.

**Comment by autarch** [comment]
This should be ready for review.

File: lib/MetaCPAN/Client/File.pm [hunk 1]
```diff
@@ -102,59 +102,164 @@ A MetaCPAN file entity object.
...
+A path you can use with the C<< MetaCPAN::Client->file >> method to get the
+file that contains POD for this module. In most cases, that will be the same
+file as that one that contains this C<module> data.
+
+=back
 
 =head2 pod_lines
 
-Array-Ref.
+An
```
Comment (mickeyn) [inline lib/MetaCPAN/Client/File.pm]: are you sure about this one?

Comment (autarch) [inline lib/MetaCPAN/Client/File.pm]: Oops, fixed and force pushed.

**Pull request merged**